content/install_workstation.md

```diff
@@ -0,0 +1,196 @@
++++
```

Comment:
We will need a redirect from install_dk

Reply:
Let me fix this in the next PR. I'll add all the redirects that are currently in Sphinx.